pyproject.toml

```diff
@@ -141,9 +141,10 @@ known_medphys = [
     "pydicom", "pylinac", "pynetdicom", "libjpeg"
 ]
 known_first_party = ["pymedphys"]
+known_experimental = ["pymedphys._experimental"]
 sections = [
     "FUTURE", "STDLIB", "THIRDPARTY", "GENERAL_SCIENTIFIC", "MACHINE_LEARNING",
-    "MEDPHYS", "FIRSTPARTY", "LOCALFOLDER"
+    "MEDPHYS", "FIRSTPARTY", "EXPERIMENTAL", "LOCALFOLDER"
```

Comment:
This section here is the only piece that was actually updated. Everything else came from running `poetry run pre-commit run --all-files`.